content/_data/site.yaml

```diff
@@ -19,4 +19,5 @@ nav:
   - url: /open-source/
   - url: /blog/
   - url: /birds/
+  - url: /jobs/
```

Comment:
We might want to review/edit the OddBirding Guide linked from this page? There are a few items there that seem out of date, esp during COVID.

Reply:
I made a revision suggestion to add `(For the safety of our birds & our communities, due to the COVID pandemic our retreats are currently being conducted remotely.)` to the Oddbirding Guide here: https://github.com/oddbird/oddleventy/pull/134/files